hack/ci/e2e-k8s.sh

```diff
@@ -107,17 +114,25 @@ create_cluster() {
       \"logging-format\": \"${KUBELET_LOG_FORMAT}\""
   fi
 
-  # JSON map injected into featureGates config
-  feature_gates="{}"
-  # --runtime-config argument value passed to the API server
-  runtime_config="{}"
+  # JSON or YAML map injected into featureGates config
+  feature_gates="${FEATURE_GATES:-{\}}"
+  # --runtime-config argument value passed to the API server, again as a map
+  runtime_config="${RUNTIME_CONFIG:-{\}}"
 
   case "${GA_ONLY:-false}" in
   false)
-    feature_gates="{}"
-    runtime_config="{}"
+    :
```

Comment:
puff, you make me learn things I wish I didn't know they existed 😄  https://stackoverflow.com/a/3224910

Reply:
Oh no, you incentivised me to click it, now I also wish to forget this.